queries.sql

```diff
@@ -38,4 +47,13 @@ SELECT * FROM animals;
 UPDATE animals SET weight_kg= weight_kg * -1 WHERE weight_kg < 0;
 SELECT * FROM animals;
 
-COMMIT;
\ No newline at end of file
+COMMIT;
+
+-- queies
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT COUNT(escape_attempts) as count, neutered FROM animals GROUP BY neutered ORDER BY count DESC LIMIT 1;
+SELECT MAX(weight_kg) as max_weight_in_kg, MIN(weight_kg) as min_weight_in_kg, species FROM animals GROUP BY species;
+SELECT AVG(escape_attempts) as average_escape_attempts, species FROM animals WHERE date_of_birth > '1990-01-01' AND date_of_birth < '2000-01-01' GROUP BY species;
```

Comment:
- [ ] Ouch! this is a close one but the years `1990` and `2000` are inclusive in the query at line `59` so I'd suggest using `>=` and `<=` like `...WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-01-01'...`